bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/discovery/HomeAssistantDiscovery.java

```diff
@@ -63,10 +69,13 @@
  *
  * @author David Graeff - Initial contribution
  */
-@Component(service = DiscoveryService.class, configurationPid = "discovery.mqttha")
+@Component(service = DiscoveryService.class, configurationPid = "org.openhab.binding.mqtt.homeassistant", property = Constants.SERVICE_PID
```

Comment:
Just wondering if there is a new naming convention for configration pid or is this just your personal preferrence. (i don;t have an opinion on this, but when reviewing other PR's i see differences.

Reply:
I couldn't find that discovery.* showed up in the UI anywhere to actually be configurable. So I based it on other @ConfigurableComponent annotations, such as on HomekitImpl. But maybe I just don't know where to look. I agree it would be much better if there was somewhere to configure this alongside other discovery services.

Reply:
I would opt to leave it as is. `discovery.mqttha`